mantidimaging/gui/utility/qt_helpers.py

```diff
@@ -235,3 +235,13 @@ def delete_all_widgets_from_layout(lo):
         # layout and marks them for deletion)
         elif item.widget() is not None:
             item.widget().setParent(None)
+
+
+def populate_menu(menu, actions_list):
```

Comment:
```suggestion
def populate_menu(menu: QMenu, actions_list: List[QAction]):
```
This should work if you commit both, but perhaps best to do it locally